### ISSUE TITLE
Footer MVP 

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#48]
What's the minimum necessary footer to ship this thing? Most likely follow lead of USWDS: 

![image](https://cloud.githubusercontent.com/assets/1165933/19166857/ebd18afc-8bce-11e6-9b25-81dfb10a46ea.png)

**Comment by francisrupert** [comment]
Consider making use of...

``` ruby
{% if jekyll.environment == "development" %}
stuff
{% endif %}

```

**Comment by francisrupert** [comment]
closed via      d052cc6
